src/lib/bootloader/grub2base.rb

```diff
@@ -414,36 +414,39 @@ def propose_encrypted
     #
     # @return [String]
     def secure_boot_summary
-      _("Secure Boot:") + " " + (secure_boot ? _("enabled") : _("disabled")) + " " +
-        if secure_boot
-          "<a href=\"disable_secure_boot\">(" + _("disable") + ")</a>"
-        else
-          "<a href=\"enable_secure_boot\">(" + _("enable") + ")</a>"
-        end
+      link = if secure_boot
+        "<a href=\"disable_secure_boot\">(#{_("disable")})</a>"
+      else
+        "<a href=\"enable_secure_boot\">(#{_("enable")})</a>"
+      end
+
+      "#{_("Secure Boot:")} #{secure_boot ? _("enabled") : _("disabled")} #{link}"
```

Comment:
NP: this pattern is repeated several times in the code, what about defining some helper like

```ruby
def status_string(status)
  status ? _("enabled") : _("disabled")
end

# and then use in
"#{_("Secure Boot:")} #{status_string(secure_boot)} #{link}"

```

Reply:
yeap, also notice it.